### PR TITLE
update deployment docs & run fmt

### PR DESCRIPTION
**Pull request by Seburan**
# Description

- update deployment doc with instructions related to enrollment and attestations (how to bypass for local testing)
- fun fmt on dsp codebase
- run fmt on tools/

## Affected services

- [ ] Home
- [ ] News
- [ ] Shop
- [ ] Travel
- [x] DSP
- [ ] SSP
- [ ] ALL

Other:
- Documentation

### CODE REVIEW

File: services/dsp/src/public/js/dsp.js
Comment (siddharth-sahoo) [inline services/dsp/src/public/js/dsp.js:1]: nit: Single quotes are preferable.

File: services/dsp/src/public/js/private-aggregation.js
Comment (siddharth-sahoo) [inline services/dsp/src/public/js/private-aggregation.js:1]: Same as above, please don't replace. Single quotes are preferable.

**Review by siddharth-sahoo — APPROVED** [review]
Left a couple of nits: single quotes are preferable for string.

File: services/dsp/src/public/js/dsp.js
Comment (Seburan) [reply services/dsp/src/public/js/dsp.js:1]: fixed

**Pull request merged**